pillars/3_HST/cert.sls

```diff
@@ -0,0 +1,58 @@
+include:
+  - letsencrypt
+  - mysql
+  - php
+  - user.webdevs.secrets
+
+
+letsencrypt:
+  post_hooks:
+    restart_apache2.sh: /usr/sbin/service apache2 reload
+mounts:
+  - spec: /dev/xvdf
+    file: /var/www
+    type: ext4
+    opts: defaults
+    freq: 0
+    pass: 2
+states:
+  mount: {{ sls }}
+  php.mbstring: {{ sls }}
+  user.webdevs: {{ sls }}
+  wordpress.apache2: {{ sls }}
+  wordpress.git_install: {{ sls }}
+wordpress:
+  docroot: /var/www/cert
+  # Multisite
+  multisite: True
+  subdomain_install: False
+  path_current_site: /
+  site_id_current_site: 1
+  blog_id_current_site: 1
+  # for plugins and themes that are not available to be installed via composer
+  git_install:
```

Comment:
Great to see how git dependencies are being managed (not through composer `git` support like I originally assumed).